### PR TITLE
Admin

### PR DESCRIPTION
**Pull request by MrBeardface**
Edited the time zone issue that was taking place in the application.rb file.

Added a few simple tests for the dashboard method in the Admin::DashboardController

### CODE REVIEW

File: spec/controllers/admin/dashboards_controller_spec.rb [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+		it "@all_tips is assigned the total Tips count" do
+			all_tips = assigns(:all_tips)
+			expect([:all_tips].count).to be >= 1
+		end
+		it "@all_bookmarks is not empty" do
+			all_bookmarks = assigns(:all_bookmarks)
+			expect([:all_bookmarks]).to_not be_empty
+		end
+	end
+end
```
Comment (ohlincik) [inline spec/controllers/admin/dashboards_controller_spec.rb]: Make sure you add a newline at the end of the file. As you can see, the DIFF is adding a hint in there as well.

File: spec/controllers/admin/dashboards_controller_spec.rb [hunk 2]
```diff
@@ -1,5 +1,27 @@
 require 'rails_helper'
 
-RSpec.describe Admin::DashboardsController, :type => :controller do
```
Comment (ohlincik) [inline spec/controllers/admin/dashboards_controller_spec.rb]: Are you not using RSpec for the tests anymore or did the syntax change?

**Comment by ohlincik** [comment]
I assume that the tests run and do what they are supposed to do ;-) You can go ahead and merge.

File: spec/controllers/admin/dashboards_controller_spec.rb (hunk 2 above)
Comment (MrBeardface) [inline spec/controllers/admin/dashboards_controller_spec.rb]: Syntax change

**Comment by ohlincik** [comment]
Ok. I tried to run the specs over the weekend and most of them now fail because we took out the Devise modules and a lot of the specs were testing functionality connected with user sign-up and sign-in.

As the next step we should clean up the defunct specs and add new ones to test the new functionality.

You can go ahead and merge this pull request.

**Pull request merged**